lib/utils/ember.js

```diff
@@ -129,6 +129,7 @@ function isEmberCoreModule(node, module, filePath) {
 
 function isEmberObject(node) {
   return (
+    node.callee &&
```

Comment:
Why do we need to add this check? If we do need to add it, probably better to check `utils.isCallExpression(node)` instead.